frontend/src/components/cycleCount/mobile/MobileCycleCountBatch.jsx

```
@@ -0,0 +1,350 @@
+      setCompleting(false);
+    }
+  };
+  
+  // Filter items based on search term
+  const filteredItems = items ? items.filter(item => {
+    const searchLower = searchTerm.toLowerCase();
+    
+    // Search in tool properties
+    if (item.item_type === 'tool' && item.tool) {
+      return (
+        (item.tool.tool_number && item.tool.tool_number.toLowerCase().includes(searchLower)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)